### DISCUSSION TITLE
Manager Maximized, how to make it floating menu again?

### DISCUSSION THREAD

**Discussion by SebAnt333** [Comfy-Org/ComfyUI-Manager#171] [Q&A]
My ComfyUI Manager menu is maximized and stuck to the right side of the screen. How can I make it a smaller floating menu again?

**Comment by ltdrdata** [comment]
It seems like you're referring to the Main menu, not the Manager menu. That happens when the resolution is too low. If you're experiencing this issue on the same computer, it might be that your browser is zoomed in, so please zoom out. You can hover over the menu, hold down ctrl, and scroll the mouse wheel to adjust.
